### ISSUE TITLE
No module named 'laspy'

### ISSUE THREAD

**Issue by kazuf3** [ceptontech/cepton_sdk_redist#10]
```
$ cepton_export.py --capture_path lidar.pcap ./
Traceback (most recent call last):
  File "/home/kazuhiro/.local/bin/cepton_export.py", line 16, in <module>
    import cepton_sdk.export
  File "/home/kazuhiro/.local/lib/python3.6/site-packages/cepton_sdk/export.py", line 6, in <module>
    import laspy
ModuleNotFoundError: No module named 'laspy'

$ uname -a
Linux unit.svil.tech 5.3.0-26-generic #28~18.04.1-Ubuntu SMP Wed Dec 18 16:40:14 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

It assumes user has 'laspy' module but pip3 didn't take care of it.

**Comment by kazuf3** [comment]
as well as `plyfile`, `transforms3d`. And above command deleted my `/home` ......

**Comment by liaody** [comment]
This is caused by missing requirements in the package. We have a fix for this that will be released in next week. Please for now just do "pip3 install ..." to work around it.

**Comment by liaody** [comment]
Fix released in 1.17.7
